src/_P076_HLW8012.ino

```diff
@@ -49,8 +45,23 @@ unsigned long p076_timer{};
 
 float p076_hcurrent{};
 float p076_hvoltage{};
-float p076_hpower{};
+float p076_hpowerActive{};
+float p076_hpowerReactive{};
+float p076_hpowerApparent{};
 float p076_hpowfact{};
+float p076_henergy{};
+
+# define P076_NR_OUTPUT_VALUES   4
+# define P076_NR_OUTPUT_OPTIONS  7
+# define P076_QUERY1_CONFIG_POS  1
+# define P076_QUERY1          PCONFIG(1)
+# define P076_QUERY2          PCONFIG(2)
+# define P076_QUERY3          PCONFIG(3)
+# define P076_QUERY4          PCONFIG(4)
```

Comment:
PCONFIG(4) is already used.
Have to check the others too, as it seems odd those were not yet used.

Reply:
Hi,
you are right, PCONFIG 4, 5, 6 and 7 are already used.
I change the new ones to 8, 9 ,10 and 11.

Reply:
> I change the new ones to 8, 9 ,10 and 11.

The max. is 7 as there are only 8 of those.
Maybe 0 ... 3 are unused?

Reply:
Okay,
I changed to 0-3